pkg/sink/kafka/claimcheck/claim_check.go

```diff
@@ -102,7 +101,7 @@ func (c *ClaimCheck) WriteMessage(ctx context.Context, key, value []byte, fileNa
 
 // FileNameWithPrefix returns the file name with prefix, the full path.
 func (c *ClaimCheck) FileNameWithPrefix(fileName string) string {
-	return filepath.Join(c.storage.URI(), fileName)
+	return c.storage.URI() + "/" + fileName
```

Comment:
Please add unit test to cover this function.

Reply:
if the configure end with "/", whether it is still work?